releasenotes/notes/fix-dbapi-rowcount-guard-6c6455278f15d262.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    dbapi: avoid type error when setting tag for row count.
```

Comment:
Probably worth pointing out that this only affects non-complaint db libraries (just so we don't scare/send anyone down a rabbit hole)